src/main/java/org/openrewrite/java/testing/junit5/RemoveTryCatchBlocksFromUnitTests.java

```diff
@@ -53,48 +53,60 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
         return new RemoveTryCatchBlocksFromUnitsTestsVisitor();
     }
 
-    private static class RemoveTryCatchBlocksFromUnitsTestsVisitor extends JavaIsoVisitor<ExecutionContext> {
+    private static class RemoveTryCatchBlocksFromUnitsTestsVisitor extends JavaVisitor<ExecutionContext> {
         @Override
         public J.MethodDeclaration visitMethodDeclaration(J.MethodDeclaration md, ExecutionContext ctx) {
+            if (md.getBody() == null) {
```

Comment:
Before returning anything, you should add a `super.visitMethodDeclaration()` call, because the `J.Try` could be nested inside a block or loop or so.

Reply:
Ah ok, so that will not interfere with the cursor messaging? I can have a super call both at the start and at the last return?

Reply:
Maybe I didn't check carefully enough. I just think we need to make sure it gets called at least once per code path in this recipe.